rotkehlchen/exchanges/manager.py

```diff
@@ -27,23 +28,24 @@
 
 # Exchanges for which we have supported modules
 SUPPORTED_EXCHANGES = [
-    Location.KRAKEN,
-    Location.POLONIEX,
-    Location.BITTREX,
-    Location.BITMEX,
     Location.BINANCE,
+    Location.BINANCEUS,
+    Location.BITCOINDE,
+    Location.BITFINEX,
+    Location.BITMEX,
+    Location.BITPANDA,
+    Location.BITSTAMP,
+    Location.BITTREX,
     Location.COINBASE,
     Location.COINBASEPRO,
     Location.GEMINI,
-    Location.BITSTAMP,
-    Location.BINANCEUS,
-    Location.BITFINEX,
-    Location.BITCOINDE,
     Location.ICONOMI,
+    Location.KRAKEN,
     Location.KUCOIN,
     Location.FTX,
+    Location.FTXUS,
```

Comment:
why did you reorder those?

Reply:
To have them by lex order as we do in other places

Reply:
We don't have it in lex order anywhere. Lex order is only at imports done automatically by isort. There is no need for this change, please don't waste time with stuff like that.

Plus ... ehm ... it's not lex order :sweat:

Reply:
<img src="https://media1.giphy.com/media/55itGuoAJiZEEen9gg/giphy.gif"/>